coalib/collecting/Collectors.py

```diff
@@ -184,15 +184,16 @@ def filter_section_bears_by_languages(bears, languages):
                         any language from languages.
     """
     new_bears = {}
-    languages = set(x.lower() for x in languages)
+    # All bears with "all" languages supported shall be shown
+    languages = set(x.lower() for x in languages) | {'all'}
     for section in bears.keys():
         filtered = []
         for bear in bears[section]:
             bear_languages = getattr(bear, 'LANGUAGES', tuple())
             if isinstance(bear_languages, str):
                 bear_languages = (bear_languages,)
             supported_languages = set(x.lower() for x in bear_languages)
-            if supported_languages & languages or 'all' in supported_languages:
+            if supported_languages & languages:
```

Comment:
We can combine 195 and 196 into one line since we don't use `supported_languages` anywhere else